crates/wallet/README.md

```diff
@@ -8,33 +8,33 @@
   </p>
 
   <p>
-    <a href="https://crates.io/crates/bdk"><img alt="Crate Info" src="https://img.shields.io/crates/v/bdk.svg"/></a>
+    <a href="https://crates.io/crates/bdk_wallet"><img alt="Crate Info" src="https://img.shields.io/crates/v/bdk_wallet.svg"/></a>
     <a href="https://github.com/bitcoindevkit/bdk/blob/master/LICENSE"><img alt="MIT or Apache-2.0 Licensed" src="https://img.shields.io/badge/license-MIT%2FApache--2.0-blue.svg"/></a>
     <a href="https://github.com/bitcoindevkit/bdk/actions?query=workflow%3ACI"><img alt="CI Status" src="https://github.com/bitcoindevkit/bdk/workflows/CI/badge.svg"></a>
     <a href="https://coveralls.io/github/bitcoindevkit/bdk?branch=master"><img src="https://coveralls.io/repos/github/bitcoindevkit/bdk/badge.svg?branch=master"/></a>
-    <a href="https://docs.rs/bdk"><img alt="API Docs" src="https://img.shields.io/badge/docs.rs-bdk-green"/></a>
+    <a href="https://docs.rs/bdk_wallet"><img alt="API Docs" src="https://img.shields.io/badge/docs.rs-bdk_wallet-green"/></a>
     <a href="https://blog.rust-lang.org/2022/08/11/Rust-1.63.0.html"><img alt="Rustc Version 1.63.0+" src="https://img.shields.io/badge/rustc-1.63.0%2B-lightgrey.svg"/></a>
     <a href="https://discord.gg/d7NkDKm"><img alt="Chat on Discord" src="https://img.shields.io/discord/753336465005608961?logo=discord"></a>
   </p>
 
   <h4>
     <a href="https://bitcoindevkit.org">Project Homepage</a>
     <span> | </span>
-    <a href="https://docs.rs/bdk">Documentation</a>
+    <a href="https://docs.rs/bdk_wallet">Documentation</a>
   </h4>
 </div>
 
-## `bdk`
+### Wallet
```

Comment:
All the other crates' `README.md` starts with a H1:

- `chain`: https://github.com/bitcoindevkit/bdk/blob/80e190b3e734a46ba73925d381fe4f1a9941bf2a/crates/chain/README.md?plain=1#L1-L3
- `bitcoind_rpc`: https://github.com/bitcoindevkit/bdk/blob/80e190b3e734a46ba73925d381fe4f1a9941bf2a/crates/bitcoind_rpc/README.md?plain=1#L1-L3
- `electrum`: https://github.com/bitcoindevkit/bdk/blob/80e190b3e734a46ba73925d381fe4f1a9941bf2a/crates/electrum/README.md?plain=1#L1-L3
- `esplora `: https://github.com/bitcoindevkit/bdk/blob/80e190b3e734a46ba73925d381fe4f1a9941bf2a/crates/esplora/README.md?plain=1#L1-L3
- `file_store `: https://github.com/bitcoindevkit/bdk/blob/80e190b3e734a46ba73925d381fe4f1a9941bf2a/crates/file_store/README.md?plain=1#L1-L3
- `testenv`: https://github.com/bitcoindevkit/bdk/blob/80e190b3e734a46ba73925d381fe4f1a9941bf2a/crates/testenv/README.md?plain=1#L1-L3

Why `wallet` starts with a H3?
I think we should change it to a H1 like the other crates.

Reply:
Good cleanup 🏕️, fixed.